contrib/ec2/setup-proxy-protocol.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+#
+# Usage: setup-proxy-protocol <cloudformation stack name>
```

Comment:
Angle brackets implies that its mandatory. Could you change it to `setup-proxy-protocol.sh [STACK]`, please?